src/components/activity/ActivityList.js

```diff
@@ -0,0 +1,53 @@
+import styled from 'styled-components';
+
+function ActivityList({activities, onHandleDelete}) {
+  const handleDelete = event => {
+    const activityId = event.target.dataset.id;
+    onHandleDelete(activityId);
+  };
+  return (
+    <>
+      {activities.map(activity => (
+        <div key={activity.id}>
+          <StyledActivitiList>
+            <p>{activity.activity}</p>
+            <p>{activity.startTime}</p>
+            <p>{activity.endTime}</p>
+            <StyledButton onClick={handleDelete} data-id={activity.id}>
+              Eintrag löschen
+            </StyledButton>
+          </StyledActivitiList>
+        </div>
+      ))}
+    </>
+  );
+}
+
+const StyledActivitiList = styled.div`
```

Comment:
Change the `div` to an `ul` and maybe you should add styling:
```suggestion
const StyledActivitiList = styled.ul`
list-style: none;
```